tgui/packages/common/setting_config.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * This helps to let players have different settings upto each codebase.
+ * The key for backend (check storage.js) is determined by a value declared here
+ *
+ * @file
+ * @copyright 2023 EvilDragonfiend
+ * @license MIT
+ */
+
+import { storage } from "./storage";
+
+
+
+// This key is important as a configurable setting.
+// This key is recommended to be different up to each codebase
+export const PREF_ADDITION_KEY = 'tgstation';
```

Comment:
Wouldn't this be better as a config(in config.txt), if that's technically possible?

Reply:
The goal with this is to make it as easy as possible for downstreams to edit these.

Reply:
I know and I agree. ngl I wanted to do that, but my knowledge of javascript isn't very good to improve more than this, and I am kinda burned out for more.

Reply:
hmm, actually I got an idea

Reply:
addressed